Userland/Libraries/LibWeb/Geometry/DOMMatrixReadOnly.h

```diff
@@ -129,5 +129,6 @@ class DOMMatrixReadOnly : public Bindings::PlatformObject {
 
 WebIDL::ExceptionOr<void> validate_and_fixup_dom_matrix_2d_init(DOMMatrix2DInit& init);
 WebIDL::ExceptionOr<void> validate_and_fixup_dom_matrix_init(DOMMatrixInit& init);
+WebIDL::ExceptionOr<Tuple<Gfx::DoubleMatrix4x4, bool>> parse_dom_matrix_init_string(JS::Realm& realm, String const& transform_list);
```

Comment:
Could we put this into a named struct instead of a tuple? More readable at the call site and the declaration